### PR TITLE
Fixes for 0.15.0

### PR DESCRIPTION
**Pull request by koenderks**
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1217
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1198
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1256
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1271
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1270
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1218
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1231
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1220
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1291
- Implements smoother distributions in prior (and posterior) plots

### CODE REVIEW

**Comment by JorisGoosen** [comment]
The new image in help functionality should be in the latest versions of JASP, so it ought to work if you made the correct changes.

File: R/auditClassicalNumberBunching.R [hunk 1]
```diff
@@ -70,9 +70,36 @@ auditClassicalNumberBunching <- function(jaspResults, dataset, options, ...){
...
+  
+  # To-do: make a faster way to find these decimals
+  decimalVec <- NULL
+  for (i in 1:length(dataset[[options[["values"]]]])) {
+    decimal <- nchar(strsplit(as.character(dataset[[options[["values"]]]][i]), "\\.")[[1]][2])
+    if (is.na(decimal)) 
+      decimal <- 0
+    decimalVec[i] <- decimal
+  }
+  print(decimalVec)
```
Comment (Kucharssim) [inline R/auditClassicalNumberBunching.R:94]:
```suggestion
```

File: inst/help/auditbayesianplanning.md [hunk 2]
```diff
@@ -176,7 +178,7 @@ R Packages
 References
 -------
 
-Derks, K (2021). jfa: Bayesian and Classical Audit Sampling. R package version 0.5.1.
+Derks, K (2021). jfa: Bayesian and Classical Audit Sampling. R package version 0.5.3.
```
Comment (Kucharssim) [inline inst/help/auditbayesianplanning.md:181]:
```suggestion
Derks, K (2021). jfa: Bayesian and Classical Audit Sampling. R package version 0.5.5.
```
right?

File: inst/help/auditselection.md [hunk 3]
```diff
@@ -114,4 +116,4 @@ References
 
 AICPA (2017). <i>Audit Guide: Audit Sampling</i>. American Institute of Certified Public Accountants.
 
-Derks, K (2021). jfa: Bayesian and Classical Audit Sampling. R package version 0.5.1.
\ No newline at end of file
+Derks, K (2021). jfa: Bayesian and Classical Audit Sampling. R package version 0.5.3.
```
Comment (Kucharssim) [inline inst/help/auditselection.md:119]:
```suggestion
Derks, K (2021). jfa: Bayesian and Classical Audit Sampling. R package version 0.5.5.
```

**Review by Kucharssim — APPROVED** [review]
Looks great! The issues appear to be fixed, code looks fine. I suggested some small changes that are not really consequential.

**Comment by koenderks** [comment]
Thank you for spotting those @Kucharssim! I will await the unit tests and then merge these changes.

**Pull request merged**